recipes/diligent-fx/all/conanfile.py

```diff
@@ -101,6 +101,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.includedirs.append(os.path.join("include"))
```

Comment:
this is useless: https://docs.conan.io/en/latest/reference/conanfile/methods.html#cpp-info

Reply:
actually, I just wanted to demonstrate this wired behaviour of conan or cmake:
in test package I have:
```
#include <DiligentFx/Components/interface/ShadowMapManager.hpp>
#include <DiligentFX/GLTF_PBR_Renderer/interface/GLTF_PBR_Renderer.hpp>
```

and in cache I also have them both:
```
./cci.20220219/_/_/package/cfd286a226097c9874fc0577e8929ba12cc0fd49/include/DiligentFx/GLTF_PBR_Renderer/interface/GLTF_PBR_Renderer.hpp
./cci.20220219/_/_/package/cfd286a226097c9874fc0577e8929ba12cc0fd49/include/DiligentFx/Components/interface/ShadowMapManager.hpp
```

But on some reson one include works and another - don't!

Reply:
you've got a casing issue here on the last letter of `DiligentFX`